Test-Data/ExampleRepos/3.0/servercore/amd64/Dockerfile

```diff
@@ -1,4 +1,4 @@
+  # escape=`
```

Comment:
This is usually used on Windows docker images and is stylistic choice to match PowerShell's escape character.